### PR TITLE
OP-1147 Upgrade to Spring Boot 3.2.1

### PR DESCRIPTION
**Pull request by Th0rst3nH**
Core part (OP-1148)

- [x] fixed numerous tests
- [x] switched javax.* to jakarta.*

### CODE REVIEW

File: src/main/java/org/isf/sms/service/SmsSenderOperations.java [hunk 1]
```diff
@@ -21,34 +21,39 @@
  */
 package org.isf.sms.service;
 
-import java.util.List;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/sms/service/SmsSenderOperations.java:24]: Please honor the projects ordering of imports; see https://github.com/flowable/flowable-engine/blob/main/ide-settings/eclipse/Preferences-Java-CodeStyle-OrganizeImports.importorder

File: src/main/java/org/isf/utils/sms/SetupGSM.java [hunk 2]
```diff
@@ -21,29 +21,30 @@
...
 import org.isf.sms.providers.gsm.GSMParameters;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.boot.web.server.PortInUseException;
 
-import gnu.io.CommPortIdentifier;
-import gnu.io.SerialPort;
-import gnu.io.SerialPortEvent;
-import gnu.io.SerialPortEventListener;
+import javax.swing.*;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/utils/sms/SetupGSM.java:40]: Please honor the project's standards about not using package level, wild card, imports.  Thanks.

File: src/test/java/org/isf/dicom/TestFileSystemDicomManager.java [hunk 3]
```diff
@@ -40,19 +32,28 @@
...
 import org.springframework.context.ApplicationContext;
 import org.springframework.util.FileSystemUtils;
 
+import java.io.File;
+import java.io.FileReader;
+import java.lang.reflect.Field;
+import java.text.ParseException;
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (dbmalkovsky) [inline src/test/java/org/isf/dicom/TestFileSystemDicomManager.java:54]: the project standard is to put static imports first.

File: src/test/java/org/isf/operation/Tests.java [hunk 4]
```diff
@@ -995,8 +994,8 @@ public void testOperationRowEquals() throws Exception {
 		Operation operation = testOperation.setup(operationType, true);
 		OperationRow operationRow = testOperationRow.setup(operation, false);
 
+		assertThat(operationRow).isEqualTo(operationRow);
 		assertThat(operationRow)
-				.isEqualTo(operationRow)
```
Comment (dbmalkovsky) [inline src/test/java/org/isf/operation/Tests.java:999]: why was the chaining of all the tests undone?

File: src/main/java/org/isf/sms/service/SmsSenderOperations.java (hunk 1 above)
Comment (Th0rst3nH) [reply src/main/java/org/isf/sms/service/SmsSenderOperations.java:24]: Sorry forgot to add the [OpenHospital-code-style-configuration.xml](https://github.com/informatici/openhospital-core/blob/develop/.ide-settings/idea/OpenHospital-code-style-configuration.xml) , should be fixed

File: src/main/java/org/isf/utils/sms/SetupGSM.java (hunk 2 above)
Comment (Th0rst3nH) [reply src/main/java/org/isf/utils/sms/SetupGSM.java:40]: Sorry forgot to add the [OpenHospital-code-style-configuration.xml](https://github.com/informatici/openhospital-core/blob/develop/.ide-settings/idea/OpenHospital-code-style-configuration.xml) , should be fixed

File: src/test/java/org/isf/dicom/TestFileSystemDicomManager.java (hunk 3 above)
Comment (Th0rst3nH) [reply src/test/java/org/isf/dicom/TestFileSystemDicomManager.java:54]: Sorry forgot to add the [OpenHospital-code-style-configuration.xml](https://github.com/informatici/openhospital-core/blob/develop/.ide-settings/idea/OpenHospital-code-style-configuration.xml) , should be fixed

File: src/test/java/org/isf/operation/Tests.java (hunk 4 above)
Comment (dbmalkovsky) [reply src/test/java/org/isf/operation/Tests.java:999]: Yes it should always be true but this verifies it and makes helps to increase the test coverage numbers 😏

**Comment by dbmalkovsky** [comment]
Closing this PR and replacing it with one more up todate.

**Pull request closed without merging**